### PR TITLE
https://github.com/riesenia/cakephp-duplicatable/issues/39

### PR DESCRIPTION
**Pull request by r34117y**

### CODE REVIEW

**Comment by ADmad** [comment]
Tests required.

**Comment by r34117y** [comment]
> Tests required.

Ok, I will update PR, but rather not today.

File: src/Model/Behavior/DuplicatableBehavior.php [hunk 1]
```diff
@@ -176,7 +178,7 @@ protected function _getContain()
     protected function _modifyEntity(EntityInterface $entity, $object)
     {
         // belongs to many is tricky
-        if ($object instanceof BelongsToMany) {
+        if ($object instanceof BelongsToMany && ! $this->getConfig('preserveJoinData')) {
```
Comment (challgren) [inline src/Model/Behavior/DuplicatableBehavior.php:181]: Any reason for the space after the `!`

Comment (r34117y) [reply src/Model/Behavior/DuplicatableBehavior.php:181]: Easier to read in my opinion. I tend to lose "!", while in "!$" context. I was also convinced that it's part of PSR Coding Style Guide, but can't find it.

**Pull request merged**